runtime/oti/j9javaaccessflags.h

```diff
@@ -67,7 +67,7 @@
 #define J9AccClassReferenceSoft 0x20000000
 #define J9AccClassReferenceWeak 0x10000000
 #define J9AccClassRomToRamMask 0xF3000000
-#define J9AccClassUnsafe 0x40000
+#define J9AccClassUnsafe 0x40000 /* obsolete - do not use */
```

Comment:
Typcially we define J9AccClassUnused0x4000 or something like that. If you haven't deleted the constant, you can't prove you found all the references.

Reply:
It's still referenced in DDR code for reading older core files.